djl-serving/wlm/flink-sentiment-analysis/build.gradle

```diff
@@ -14,9 +14,9 @@ repositories {
 
 dependencies {
     implementation 'org.apache.flink:flink-streaming-java_2.11:1.11.1'
-    implementation platform("ai.djl:bom:0.19.0")
+    implementation platform("ai.djl:bom:0.20.0")
     implementation "ai.djl:api"
-    implementation "ai.djl.serving:wlm:0.19.0"
+    implementation "ai.djl.serving:wlm:0.20.0"
```

Comment:
We didn't publish serving yet, this PR has to wait until serving is published next week